### ISSUE TITLE
Support for Mocap Mode (Anchor to head)

### ISSUE THREAD

**Issue by ZRock35** [ExpiredPopsicle/SnekStudio#45]
If possible, a way to anchor to head rather than hip? I use SlimeVR for VTubing in full body in what is called "Mocap Mode", and it has a feature that lets you move freely around in the scene, anchoring to head instead of hip. If this is already existent, then disregard this issue, however, if help is needed, I am willing to assist in any testing! It would vastly improve the experience.

**Comment by ExpiredPopsicle** [comment]
The MediaPipe based tracking is already based on the head location, with hips following that.

How would this be different from existing behavior?

**Comment by ZRock35** [comment]
Yes, it works fine passing VMC with head lock on through on VSeeFace and Warudo, as well as Blender and Unity, I am able to walk around the scenes okay in those, so it must be that!

**Comment by ExpiredPopsicle** [comment]
Hey, I know this bug has been open for a while, but I'll have a better chance to tackle it now that I have my own SlimeVR set and can see the OSC data coming in from the SlimeVR driver itself.
